src/backend/marsha/core/tests/test_api_timed_text_track.py

```diff
@@ -189,15 +189,30 @@ def test_api_timed_text_track_read_list_anonymous(self):
         self.assertEqual(response.status_code, 401)
 
     def test_api_timed_text_track_read_list_token_user(self):
-        """A token user associated to a video is not able to read a list of timed text tracks."""
-        timed_text_track = TimedTextTrackFactory()
+        """A token user associated to a video is able to read a list of timed text tracks."""
+        timed_text_track_cc = TimedTextTrackFactory(
+            id="e996995f-f981-4657-b796-8842ac20f20d", mode="cc"
```

Comment:
approving this PR. If you want to simplify I think it would work the same letting the `id` and `mode` fields be set randomly by the factory.